nilearn/_utils/niimg.py

```diff
@@ -0,0 +1,157 @@
+"""
+Neuroimaging file input and output.
+"""
+# Author: Gael Varoquaux, Alexandre Abraham, Philippe Gervais
+# License: simplified BSD
+
+import copy
+import gc
+import collections
+from six import string_types
+
+import numpy as np
+import nibabel
+
+from .numpy_conversions import as_ndarray
+
+
+def _safe_get_data(img):
+    """ Get the data in the image without having a side effect on the
```

Comment:
>     1.3.0 is targetted.

> then README.rst and nilearn/version.py should be updated. We are currently
> requiring nibabel 1.1.0 there.

OK, my answer was wrong. The question was: is it version 1.3.0 or version
2.0 that is targetted. I replied 1.3.0 but I should have replied 2.0.